katcp/test/test_resource_client.py

```diff
@@ -16,7 +24,7 @@
 import tornado
 import mock
 
-from thread import get_ident as get_thread_ident
+from _thread import get_ident as get_thread_ident
```

Comment:
It looks like the `_thread` module is provided by the `future` package, so yes, this is presumably the right thing to do. In Python 3 it also seems to be available as `threading.get_ident` but that doesn't work on Python 2.